runtime/descriptor.go

```diff
@@ -117,5 +119,31 @@ func makeStructFieldDescriptor(t *Type, fieldName, propertyName string) *Object
 		}
 		return ret, raised
 	}
+	if writtable {
+		setterFunc := func(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
+			ret := None
+			var raised *BaseException
+			if raised = checkFunctionArgs(f, fieldName, args, ObjectType, ObjectType); raised == nil {
+				self := args[0]
+				newvalue := args[1]
+				if !newvalue.isInstance(nativeFieldType) {
+					format := "descriptor '%s' for '%s' objects doesn't apply to '%s' objects"
+					raised = f.RaiseType(TypeErrorType, fmt.Sprintf(format, propertyName, t.Name(), newvalue.typ.Name()))
+				} else {
+					var converted reflect.Value
+					val := t.slots.Basis.Fn(self).FieldByIndex(field.Index)
+
+					if converted, raised = maybeConvertValue(f, newvalue, val.Type()); raised == nil {
+						val.Set(converted)
+					}
+				}
+			}
+			return ret, raised
+		}
+		return newProperty(
```

Comment:
Sure. I though that too, but was not sure if I need to type the ``setter := None``, nor that ``newBuiltinFunction`` returns an Object at the time this was coded.